seatunnel-e2e/seatunnel-connector-v2-e2e/connector-pulsar-e2e/src/test/resources/pulsar/source_earliest_to_console.conf

```diff
@@ -0,0 +1,74 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+######
+###### This config file is a demonstration of streaming processing in seatunnel config
+######
+
+env {
+  # You can set flink configuration here
+  execution.parallelism = 1
+  #execution.checkpoint.interval = 10000
+  #execution.checkpoint.data-uri = "hdfs://localhost:9000/checkpoint"
+}
+
+source {
+  Pulsar {
+    topic = "json_topic_source"
+    subscription.name = "seatunnel"
+    client.service-url = "pulsar://pulsar-e2e:6650"
+    admin.service-url = "http://pulsar-e2e:8080"
+    result_table_name = "pulsar_table"
+    format_error_handle_way = skip
+    cursor.startup.mode = "EARLIEST"
+    cursor.stop.mode = "LATEST"
+    schema = {
+      fields {
+        id = bigint
+      }
+    }
+  }
+}
+
+transform {
+}
+
+sink {
+  Console {
+    source_table_name = "pulsar_table"
+  }
+  Assert {
+    source_table_name = "pulsar_table"
+    rules = {
+      field_rules = [
```

Comment:
Check the row count?

Reply:
Check the  min value and max value  of `id` field in all rows.